about/index.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: About the Club
+---
+
+# About
+
+The University of Waterloo Amateur Radio Club is a club at the
+[University of Waterloo](https://uwaterloo.ca) for exploring
+the amateur radio hobby. The club is open to all students of the University, for both
+those who are licensed and those who are not.
+
+The club callsign is VE3UOW.
+
+# Meeting
+
+The club meets on most Friday's at 6:00PM in E2 4402. Anyone is welcome
+to attend a club meeting. If you don't know where to go,
+please [contact us]({{site.url}}/contact/) and we'll show you the way.
+
+We perform many different activities at our meetings, including:
+
+- Amateur Radio Course (for those interested in getting their license)
+- Radio Activities
+	- Amateur Radio
+		- Simplex operation
+		- Repeater operation
+		- HF operation
+	- Software Defined Radio
+		- Transmission modes
+		- Radio protocols
+			- Trunked Radio
+			- Digital Radio
+			- Marine and Air Traffic
+	- Antenna Building
+
+A license is not required to participate in most activities as licensed
+members can allow non-licensed members to operate the radio with their
+supervision.
+
+# Executive
+
+A list of executive positions will be available at the start of the
```

Comment:
Yes it was, but I couldn't remember all of the positions and who was assigned to what (remembering now that I have it from the mailing list).

In addition to the list of people in the position, I want to try and give a small description of the position (do we agree?)